src/Controller/OpenIDConnectController.php

```diff
@@ -139,12 +146,24 @@ public function main(string $id) {
   private function getOpenIdConfigurationProvider(): OpenIdConfigurationProvider {
     $pluginConfiguration = $this->plugin->getConfiguration();
 
+    try {
+      $key = $this->keyRepository->getKey($pluginConfiguration[OpenIDConnect::KEY] ?? '');
+      [
+        OidcKeyType::DISCOVERY_URL => $discoveryUrl,
```

Comment:
Why are we suddenly abbreviating `OpenIDConenct`? Is it simply just to keep the key short?